src/Cassandra/Orleans.Clustering.Cassandra/Hosting/CassandraClusteringOptions.cs

```diff
@@ -10,6 +10,20 @@ namespace Orleans.Clustering.Cassandra.Hosting;
 /// </summary>
 public class CassandraClusteringOptions
 {
+    /// <summary>
+    /// Optional default time-to-live (in seconds) for entries in the membership table, allowing defunct silo cleanup
+    /// even if a cluster is no longer running. This is currently only applied at initial table creation time and will
+    /// NOT attempt to alter a table or rows that have already been created.
+    /// </summary>
+    /// <remarks>
+    /// 0 means default TTL will be disabled.
+    /// <para />
+    /// See https://docs.datastax.com/en/cql-oss/3.3/cql/cql_reference/cqlCreateTable.html#tabProp__cqlTableDefaultTTL
+    /// <para />
+    /// Initial implementation of https://github.com/dotnet/orleans/issues/9164
+    /// </remarks>
+    public uint MembershipTableDefaultTimeToLiveSeconds { get; set; } = 0;
```

Comment:
Super nit:
```suggestion
    public uint MembershipTableDefaultTimeToLiveSeconds { get; set; }
```
The default value (here `0` for `uint`) is set by the runtime as default. So explicit setting is redundant.

Reply:
This value is intentionally zero as a default as the Cassandra implementation of setting the `default_time_to_live` to zero has the effect of disabling TTL. It may be good to keep this explicit value so show the intent that this is explicitly desired and not just happenstance that it's the language default.